x-pack/plugins/fleet/server/services/form_settings/form_settings.ts

```diff
@@ -24,40 +24,19 @@ export function _getSettingsAPISchema(settings: SettingsConfig[]): Props {
     if (!setting.api_field) {
       return;
     }
-    const defaultValueRes = setting.schema.safeParse(undefined);
-    const defaultValue = defaultValueRes.success ? defaultValueRes.data : undefined;
-    if (defaultValue) {
```

Comment:
Just a question: it's ok to remove the default value but it's not clear to me why . Is that a change in requirement or was it causing some issues?